src/frontend/src/flows/authenticate/index.ts

```diff
@@ -30,65 +30,57 @@ const pageContent = (
   userNumber?: bigint,
   derivationOrigin?: string
 ) => html` <div class="l-container c-card c-card--highlight">
+    <!-- The title is hidden but used for accessibility -->
+    <h1 class="is-hidden">Internet Identity</h1>
     <div class="c-logo">${icLogo}</div>
-    <h1 class="t-title t-title--main">Internet Identity</h1>
-    <p class="t-lead">Authenticate to service:</p>
-    <div class="c-input t-vip t-vip--small c-input--readonly">${hostName}</div>
-    ${derivationOrigin !== undefined && derivationOrigin !== hostName
-      ? derivationOriginSection(derivationOrigin)
-      : ""}
-    <p class="t-paragraph">Use Identity Anchor:</p>
+    <p class="t-lead">Authenticating to <span class="t-strong">${hostName}</span> ${
+  derivationOrigin !== undefined && derivationOrigin !== hostName
+    ? html`(an alias of <span class="t-strong">${derivationOrigin}</span>)`
+    : ""
+} using <span class="t-strong">Internet Identity</h1>.</p>
```

Comment:
you are opening a span here but closing an h1

Reply:
Good catch, thanks. Will need to work on this a bit more anyway.